[2023-10-05 14:29:18,946] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2023-10-05T14:29:18.053809+00:00 [queued]>
[2023-10-05 14:29:18,959] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.retrieve_raw_data 2023-10-05T14:29:18.053809+00:00 [queued]>
[2023-10-05 14:29:18,959] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2023-10-05 14:29:18,959] {taskinstance.py:1088} INFO - Starting attempt 1 of 6
[2023-10-05 14:29:18,959] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2023-10-05 14:29:18,976] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): retrieve_raw_data> on 2023-10-05T14:29:18.053809+00:00
[2023-10-05 14:29:18,981] {standard_task_runner.py:52} INFO - Started process 1754 to run task
[2023-10-05 14:29:18,985] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'retrieve_raw_data', '2023-10-05T14:29:18.053809+00:00', '--job-id', '107', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/weather.py', '--cfg-path', '/tmp/tmp1d0poclr', '--error-file', '/tmp/tmp93_7p4cg']
[2023-10-05 14:29:18,985] {standard_task_runner.py:77} INFO - Job 107: Subtask retrieve_raw_data
[2023-10-05 14:29:19,027] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.retrieve_raw_data 2023-10-05T14:29:18.053809+00:00 [running]> on host 3a3d0401ac26
[2023-10-05 14:29:19,078] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=retrieve_raw_data
AIRFLOW_CTX_EXECUTION_DATE=2023-10-05T14:29:18.053809+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-10-05T14:29:18.053809+00:00
[2023-10-05 14:29:19,539] {python.py:151} INFO - Done. Returned value was: None
[2023-10-05 14:29:19,552] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=retrieve_raw_data, execution_date=20231005T142918, start_date=20231005T142918, end_date=20231005T142919
[2023-10-05 14:29:19,582] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-10-05 14:29:19,600] {local_task_job.py:151} INFO - Task exited with return code 0
